backend/internal/handlers/classrooms/assignments/assignments.go

```diff
@@ -126,6 +126,14 @@ func (s *AssignmentService) acceptAssignment() fiber.Handler {
 			return err
 		}
 
+		// Remove student team's access to forked repo
+		// TODO: dynamically find student team name (KHO-177)
```

Comment:
I added a field on the classroom table to hold the name of the corresponding student team